components/buildList/js/buildList.controller.js

```diff
@@ -6,6 +6,464 @@ buildApp.controller('buildListController', ['$scope', '$q', '$interval', 'buildL
 		intervalTime = 5000,
 		timeElapsed = 0;
 
+	function init() {
+				getData();
+		$q.when(deferred.promise, function() {
+			updateBuildsInterval = $interval(updateBuildsList, intervalTime);
+		});
+	}
+
+	function getData() {
+		return buildListService.getData().then(function(response) { 
+								$scope.buildList = response.data;
+			deferred.resolve();
+		});
+
+		return;
+	}
+
+	function getData() {
+		return buildListService.getData().then(function(response) { 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=javascript%3ATrailingWhitespace)